tox.ini

```diff
@@ -17,6 +19,7 @@ deps =
     doc: Sphinx
 
 [testenv]
+basepython = python3.11
```

Comment:
Any particular reason why you are hardcoding the tests to use python 3.11? That will cause a giant headache for QA, who are using the system python in openQA.

Reply:
this is still a draft, I'm trying to find a workable path forward to get it to not use the system python. however, for SLE15 I don't see a problem with requiring python311 for the QA team. it might be more difficult for the tests on non-SLE hosts, but one problem at a time.